src/core/utils/types/instant-loan.ts

```diff
@@ -1,5 +1,5 @@
 export type InstantLoanConfigType = {
-  matchString: string;
+  match_string: string;
```

Comment:
From a code style perspective we need to decide on what formatting we want here. Personally I am fine with camelCase.

This change needs to be aligned with danskernesdigitalebibliotek/dpl-cms#290.